### PR TITLE
feat(datatable): Add searchable for each columns on datatable

### PR DESCRIPTION
**Pull request by nvtin**
This is about adding one option which is called `searchable` for each column on datatable. I have updated document as well.

### CODE REVIEW

**Comment by nvtin** [comment]
Hi @RWOverdijk I completely forgot to update the pull request about adding searchable for each column. But I can't update the old commit message to match message format. So I closed that PR and make this new pull request. Can you review and merge it? Thank you so much.

File: src/datatable.js [hunk 1]
```diff
@@ -301,7 +301,8 @@ export class DataTable {
           column   : column.property,
           label    : ucfirst(clean(column.label || column.property)),
           route    : column.route || false,
-          converter: column.valueConverters || false
+          converter: column.valueConverters || false,
+          searchable: typeof column.searchable === 'boolean' ? column.searchable : true
```
Comment (RWOverdijk) [inline src/datatable.js]: Align consecutive colons

**Review by RWOverdijk — CHANGES_REQUESTED** [review]

**Comment by RWOverdijk** [comment]
I think it's a feat and not a fix

**Comment by RWOverdijk** [comment]
@jeremyvergnas quick review?

**Review by jeremyvergnas — APPROVED** [review]
Looks good to me 👍

**Comment by nvtin** [comment]
@RWOverdijk Thanks for your feedback on this PR. I have just updated the code follow your review. Can you review it again?

**Review by RWOverdijk — APPROVED** [review]

**Comment by RWOverdijk** [comment]
If you run `gulp test` you'll see there are linting errors. I fixed those and published v0.9.0

**Pull request merged**